pkg/controllers/disruption/orchestration/metrics.go

```diff
@@ -0,0 +1,61 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package orchestration
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	crmetrics "sigs.k8s.io/controller-runtime/pkg/metrics"
+
+	"github.com/aws/karpenter-core/pkg/metrics"
+)
+
+func init() {
+	crmetrics.Registry.MustRegister(disruptionReplacementNodeClaimInitializedHistogram, disruptionReplacementNodeClaimFailedCounter, disruptionQueueDepthGauge)
+}
+
+const (
+	disruptionSubsystem    = "disruption"
+	methodLabel            = "method"
+	consolidationTypeLabel = "consolidation_type"
+)
+
+var (
+	disruptionReplacementNodeClaimInitializedHistogram = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Namespace: metrics.Namespace,
+			Subsystem: disruptionSubsystem,
+			Name:      "replacement_nodeclaim_initialized_seconds",
```

Comment:
Do we want to reason about nodes or nodeclaims in these metrics?

Reply:
NodeClaim initialization is a function of Node initialization. In this case, we're including the time it takes to create the nodeclaim, launch the instance, register + initialize the node. This more so captures the e2e process of startup, so I'm inclined to make this reason about nodeclaims. Thoughts?

Reply:
Unsure. I suppose it's sufficiently internal that we can expect users to understand the NodeClaim abstraction. Maybe worth a discussion w/ the team.